src/commands/environments/copy-layout.js

```diff
@@ -43,11 +43,13 @@ class CopyLayoutCommand extends Command {
     }
 
     if (!answers || answers.confirm === toEnvironment.name) {
-      await manager.copyLayout(fromEnvironment.id, toEnvironment.id);
-      console.log(`Environment's layout ${chalk.red(fromEnvironment.name)} successfully copied to ${chalk.red(toEnvironment.name)}.`);
-    } else {
-      logger.error(`Confirmation did not match ${chalk.red(toEnvironment.name)}. Aborted.`);
+      const copyLayout = await manager.copyLayout(fromEnvironment.id, toEnvironment.id);
```

Comment:
this should be wrapped in a try catch. You should catch error from the forestadmin-server (Like 403, 404, 500) even if 404 should not happened